spec/rails_helper.rb

```diff
@@ -65,6 +69,4 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
   config.include FactoryBot::Syntax::Methods
-
-  config.include RSpec::RedisHelper, redis: true
```

Comment:
Upon reviewing, I support using `fakeredis` for our testing environment

`fakeredis` ensures that the tests are isolated from each other. As such, `redis_helper` was removed as it does not seem to be needed.

However, there might be complex Redis usage implementation in the future that makes `fakeredis` not suitable - everything seems to be working as it should on our testing environment for now.